tests/endtoend_tests/test_endtoend.py

```diff
@@ -841,17 +834,16 @@ def test_logging_tesseract_serve(
     )
     assert res.status_code == 200, res.text
 
-    log_file = Path(tmpdir) / f"logs/run_{job_id}/tesseract.log"
+    log_file = Path(tmpdir) / f"run_{job_id}/logs/tesseract.log"
     assert log_file.exists()
 
     with open(log_file) as f:
         log_content = f.read()
     assert "Hello from apply!" == log_content.strip()
 
 
-def _prepare_mpa_test_image(dummy_tesseract_package, docker_cleanup):
-    from textwrap import dedent
-
+@pytest.fixture(scope="module")
```

Comment:
Drive-by change to reduce test runtimes (re-using images that are referenced in multiple tests)